src/api/app/views/webui/comment/_comment_field.html.haml

```diff
@@ -23,7 +24,7 @@ class: "#{form_method}-comment-form" }, data: { preview_comment_url: preview_com
       .comment-controls.mb-3
         - case form_method
         - when :post
-          = f.submit 'Add comment', class: 'btn btn-primary me-2', data: { disable_with: 'Creating comment...' }, disabled: true
+          = f.submit 'Add comment', class: 'btn btn-primary me-2', data: { disable_with: 'Creating comment...' }
```

Comment:
Bring back the `disabled: true` value and check why the js connected to this button is not working